components/Homepage/Changemaker.tsx

```diff
@@ -41,68 +47,7 @@ export const Changemaker = () => {
             change, and try again. Meet some of them.
           </Paragraph>
         </AnimateInView>
-        <Grid md={2} xxl={4} pt={7} gap="default">
-          <ChangemakerCard
-            imageSrc="https://a-us.storyblok.com/f/1005200/878x1800/523e07bc72/person.jpg"
-            heading="Yuhe Faye Wang"
-            body="Lorem Ipsum Dolar Sit"
-            animation="slideUp"
-            delay={0.2}
-          >
-            <Paragraph variant="card">
-              Lorem ipsum dolor sit amet, consectetur adipiscing elit.
-              Etiam tempor sed tellus ac vulputate. Morbi varius ornare arcu,
-              quis scelerisque urna ultricies sed. Nunc eu est leo.
-            </Paragraph>
-            <Paragraph variant="card">
-              Lorem ipsum dolor sit amet, consectetur adipiscing elit.
-              Etiam tempor sed tellus ac vulputate.
-            </Paragraph>
-          </ChangemakerCard>
```

Comment:
This whole section can now be added as nested block on the MVP Homepage in Storyblok.